packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -87,3 +88,47 @@ export function createHandler<R extends OdisRequest>(
 ): PromiseHandler<R> {
   return meteringHandler(catchErrorHandler(enabled ? handler : disabledHandler<R>))
 }
+
+export function startProxy(req: any, res: any, config: CombinerConfig) {
+  let destinationUrl
+
+  const logger = rootLogger(config.serviceName)
+  const proxy = httpProxy.createProxyServer({
+    proxyTimeout: config.phoneNumberPrivacy.odisServices.timeoutMilliSeconds,
+  })
+
+  const originalPath = req.path
+  const strippedPath = originalPath.replace(/\/combiner/, '')
+
+  switch (config.proxy.deploymentEnv) {
+    case 'mainnet':
+      // XXX (soloseng):URL may need to be updated after gen2 function is created on mainnet
+      destinationUrl =
+        'https://us-central1-celo-pgpnp-mainnet.cloudfunctions.net/combinerGen2' +
+        proxy.web(req, res, { target: destinationUrl })
+      break
+    case 'alfajores':
+      // XXX (soloseng):URL may need to be updated after gen2 function is created on alfajores
+      destinationUrl =
+        'https://us-central1-celo-phone-number-privacy.cloudfunctions.net/combinerGen2' +
+        strippedPath
+
+      proxy.web(req, res, { target: destinationUrl })
+      break
+    case 'staging':
+      destinationUrl =
+        'https://us-central1-celo-phone-number-privacy-stg.cloudfunctions.net/combinerGen2' +
+        strippedPath
+
+      proxy.web(req, res, { target: destinationUrl })
+      break
+  }
+  proxy.on('error', (_) => {
+    logger.error('Error in Proxying request to Combiner.')
+    res.status(500).json({
```

Comment:
Will this override the actual error returned by the gen2 combiner? I'd rather just pass that along to the end user if possible